vscode_extension/README.md

```diff
@@ -0,0 +1,31 @@
+# AaC VSCode Extension README
```

Comment:
We should either populate this file or delete it.

Reply:
The plugin packager will fail if no readme is present. I plan to populate it a bit.

Reply:
Added some boilerplate and initial population. We'll want to reconsider our docs structure with the new extension.